src/app/clusters/window-covering-server/window-covering-server.cpp

```diff
@@ -649,4 +764,16 @@ bool emberAfWindowCoveringClusterGoToTiltPercentageCallback(app::CommandHandler
     return true;
 }
 
+/**
+ * @brief Cluster Attribute Changed Callback
+ */
+void __attribute__((weak))
```

Comment:
Why weak?  This is where spec-defined cluster logic should go.  App logic goes in the app post-attr-change notification, which is called separately.  There's no reason for an app to override this callback....

Reply:
Yes indeed, but right now we got two separated logic. The one in all-cluster-app  and the one in Window-app used on EFR32. My goal is to merge everything into a single app but due to time constraints and amount of diff that would do i propose to do that later in next PR. Here is to implement the missing tests.
If you look at stop-motion before this was using weak and now removed. Right there is one PostAttributeChange in window-covering-server.cpp and another in ZclCallbacks.cpp